deploy/v0.0.5/kubevirt-node-labeller-ds.yaml

```diff
@@ -17,12 +17,11 @@ spec:
       serviceAccount: kubevirt-node-labeller
       containers:
       - name: kubevirt-node-labeller-sleeper
-        image: quay.io/ksimon/kubevirt-node-labeller:v0.0.5
+        image: "quay.io/ksimon/kubevirt-node-labeller:v0.0.5"
```

Comment:
Same here, we should generate the manifests automatically during release. Maintaining this will be a nightmate otherwise. But lets do this in different PR.

Reply:
this deploy manifest is automaticaly generated by: make deploy-manifest (it creates new folder with tag version + replace all tags with version from ansible_vars/vars.yaml )